### PR TITLE
Add Machine Inventory documentation in UI

### LINKED ISSUE #83: Talk about machine inventory in the UI quickstart
In the UI quickstart, we guide the user until creating a machine registration endpoint but we do not mention machine inventory.

### PR DESCRIPTION
**Pull request by juadk**
Fix #83 

Preview: https://deploy-preview-91--elemental-docs-staging.netlify.app/next/quickstart-ui#machine-inventory

### CODE REVIEW

**Review by fgiudici — APPROVED** [review]
like it!

**Review by davidcassany — APPROVED** [review]
LGTM thanks :+1:

File: docs/quickstart-ui.md [hunk 1]
```diff
@@ -114,3 +114,23 @@ You can now boot your nodes with this image and they will:
 - Register with the registrationURL given and create a per-machine `MachineInventory`
 - Install Elemental Teal to the given device
 - Reboot
+
+## Machine Inventory
+
+When nodes are booting up for the first time, they connect to Rancher Manager and a [`Machine Inventory`](https://elemental.docs.rancher.com/architecture#machineinventory) is created for each nodes.
```
Comment (kkaempf) [inline docs/quickstart-ui.md:120]:
```suggestion
When nodes are booting up for the first time, they connect to Rancher Manager and a [`Machine Inventory`](https://elemental.docs.rancher.com/architecture#machineinventory) is created for each node.
```

File: docs/quickstart-ui.md [hunk 2]
```diff
@@ -114,3 +114,23 @@ You can now boot your nodes with this image and they will:
...
+
+![Machine Inventory menu](images/quickstart-ui-machine-inventory-menu.png)
+
+Custom columns are based on `Machine Inventory Labels` which you can add when you create your `Machine Registration Endpoint`:
+
+![Machine Registration Endpoint Hardware Labels](images/quickstart-ui-registration-endpoint-hardware-labels.png)
+
+On the following screenshot, [`Hardware Labels`](https://elemental.docs.rancher.com/hardwarelabels/#hardware-labels) are used as custom columns:
+
+You can add custom columns by clicking on the three dots menu.
```
Comment (kkaempf) [inline docs/quickstart-ui.md:130]:
```suggestion
You can also add custom columns by clicking on the three dots menu.
```

File: docs/quickstart-ui.md [hunk 3]
```diff
@@ -114,3 +114,23 @@ You can now boot your nodes with this image and they will:
...
+
+![Machine Inventory custom columns](images/quickstart-ui-machine-inventory-custom-columns.png)
+
+Finally, you can also filter your `Machine Inventory` using those labels, for instance if you only want to see your AMD machines, you can filter on `CPUModel` like below:
```
Comment (kkaempf) [inline docs/quickstart-ui.md:134]:
```suggestion
Finally, you can also filter your `Machine Inventory` using those labels.
For instance if you only want to see your AMD machines, you can filter on `CPUModel` like below:
```

**Review by kkaempf — CHANGES_REQUESTED** [review]
👍🏻 otherwise 😉

**Review by kkaempf — APPROVED** [review]
looking good now !

**Pull request merged**